spec/app/ics-101-interchain-swap/README.md

```diff
@@ -74,29 +75,32 @@ $$SP_i^o = (B_i/W_i)/(B_o/W_o)$$
 
 #### Fees
 
-Traders pay swap fees when they trade with a pool. These fees can be customized with a minimum value of 0.0001% and a maximum value of 10%. 
+Traders pay swap fees when they trade with a pool. These fees can be customized with a minimum value of 0.0001% and a maximum value of 10%.
 
-The fees go to liquidity providers in exchange for depositing their tokens in the pool to facilitate trades. Trade fees are collected at the time of a swap, and goes directly into the pool, increasing the pool balance. For a trade with a given $inputToken$ and $outputToken$, the amount collected by the pool as a fee is 
+The fees go to liquidity providers in exchange for depositing their tokens in the pool to facilitate trades. Trade fees are collected at the time of a swap, and goes directly into the pool, increasing the pool balance. For a trade with a given $inputToken$ and $outputToken$, the amount collected by the pool as a fee is
 
-$$Amount_{fee} = Amount_{inputToken} * swapFee$$ 
+$$Amount_{fee} = Amount_{inputToken} * swapFee$$
 
 As the pool collects fees, liquidity providers automatically collect fees through their proportional ownership of the pool balance.
```

Comment:
need to add explanation how swap fee is defined:

in order avoid use decimals, fees would defined as basis point.   100 = 1%, that means 100/10000